cid/builtin/core/data/glue/table_columns.json

```diff
@@ -0,0 +1,254 @@
+[
```

Comment:
cid/builtin/core/data/glue/table_columns.json -> cid/builtin/core/data/glue/cur_columns.json

?

Reply:
Renamed to follow our naming convention